README.md

```diff
@@ -31,7 +31,7 @@ Once you have Docker Compose installed, you can start the stack on Linux or OSX
 ```
     export ADDITIONAL_COMPOSE_SERVICES_PATH=${PWD}/gem/lib/pupperware/compose-services
     export COMPOSE_FILE=${ADDITIONAL_COMPOSE_SERVICES_PATH}/postgres.yml:${ADDITIONAL_COMPOSE_SERVICES_PATH}/puppetdb.yml:${ADDITIONAL_COMPOSE_SERVICES_PATH}/puppet.yml
-    DNS_ALT_NAMES=host.example.com docker-compose up -d
+    PUPPET_DNS_ALT_NAMES=host.example.com docker-compose up -d
```

Comment:
Ah, thanks for the contribution @tomposmiko.

I see that you're referring to https://github.com/puppetlabs/pupperware/blob/main/gem/lib/pupperware/compose-services/puppet.yml#L16.

It might make sense to just remove those instructions altogether and focus on using an override file... but if you want to get this merged, would you mind updating the other locations in the docs to refer to `PUPPET_DNS_ALT_NAMES`?

Reply:
I'm OK with your aolution, feel free to take it over.